.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ locals:
 executors:
   go:
     docker:
-      - image: docker.mirror.hashicorp.services/circleci/golang:1.13.15
+      - image: docker.mirror.hashicorp.services/circleci/golang:1.16.5
```

Comment:
packngo requires go v1.15+

I can remove this line if #200 (or similar) is merged first.